usage/tips.md

```diff
@@ -13,7 +13,7 @@ You can implement a full border for Yazi via the UI plugin.
 
 <img src={useBaseUrl("/img/full-border.png")} width="600" />
 
-Copy the preset [`manager` component](https://github.com/sxyazi/yazi/blob/main/plugin/preset/components/manager.lua) to any place, for example `~/.config/yazi/ui.lua`, then apply the following patch:
+Copy the preset [`manager` component](https://github.com/sxyazi/yazi/blob/fc0057617e06ad08b58ead8f8fa6b030916718d8/yazi-plugin/preset/components/manager.lua) to any place, for example `~/.config/yazi/ui.lua`, then apply the following patch:
```

Comment:
That link just goes to a snapshot and won't be up to date if any changes are made. Why not [`https://github.com/sxyazi/yazi/blob/main/yazi-plugin/preset/components/manager.lua`](https://github.com/sxyazi/yazi/blob/main/yazi-plugin/preset/components/manager.lua)?

Reply:
whoops, thank you. will update.